R/step-subset-slice.R

```diff
@@ -55,8 +55,9 @@
 #' # physical weight of the cars, so heavy cars are more likely to get
 #' # selected
 #' dt %>% slice_sample(weight_by = wt, n = 5)
-slice.dtplyr_step <- function(.data, ...) {
+slice.dtplyr_step <- function(.data, ..., .by = NULL) {
   dots <- capture_dots(.data, ..., .j = FALSE)
```

Comment:
Might want to call `dplyr:::check_dot_by_typo(...)` here too.

Reply:
Looks like this is called in the generic in `dplyr` so the error works in `dtplyr`

Reply:
Oh yeah, of course.